src/PlanView/SimpleItemMapVisual.qml

```diff
@@ -115,4 +128,32 @@ Item {
             opacity:        _root.opacity
         }
     }
+
+    QGCMapCircleVisuals {
```

Comment:
This needs to be add/removed dynamically as needed like the rest of the code. Otherwise all map items carry the cost of map circles even when they are not using them